kitchen-tests/cookbooks/rspec/templates/run-chef-rspec

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Not that it totally matters but why a template if it's static?

Reply:
i always use templates because in the future it might not be static.